fuse/nodefs/fsops.go

```diff
@@ -70,6 +70,8 @@ func (c *FileSystemConnector) lookupMountUpdate(out *fuse.Attr, mount *fileSyste
 
 // internalLookup - execute a lookup without affecting NodeId reference counts
 func (c *FileSystemConnector) internalLookup(out *fuse.Attr, parent *Inode, name string, header *fuse.InHeader) (node *Inode, code fuse.Status) {
+	c.forgetMu.RLock()
```

Comment:
Oh well. I think we additionally need a per-inode forgetMu.
But first, please take a look at https://github.com/rfjakob/go-fuse/commit/ab4f2be05c683b6d8090b3b19106be9e401e488b and tell me if you think nulling fsInode is correct here. I think it is, but it exposes lots of bugs. I can trigger NPEs in a few iterations of fsstress.